source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1367,7 +1395,11 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
       ENVOY_STREAM_LOG(debug, "Sending immediate response", *decoder_callbacks_);
       processing_complete_ = true;
       onFinishProcessorCalls(Grpc::Status::Ok);
-      closeStream();
+      if (config_->gracefulGrpcClose()) {
```

Comment:
Should we do graceful close for closeStream() call line: 1430 and 1441?

Reply:
See comment above